releasenotes/notes/submit-evaluation-for-01096d803d969e3e.yaml

```diff
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    LLM Observability: This introduces the ability to submit custom evaluations joined to a span by a tag key-value pair using the
+                      `LLMObs.submit_evaluation_for()` method. The tag key-value pair is expected to uniquely a span.
```

Comment:
```suggestion
    LLM Observability: This introduces the `LLMObs.submit_evaluation_for()` method, which provides the ability to submit and correlate custom evaluations to a span by a tag key-value pair present on the span rather than using span and trace IDs.
```
I'm not sure this is the best wording, but I think it would be a better idea to be more verbose about what exactly joining with custom keys means to a user. I would probably avoid the terms `joining` and rather use `correlating` here, feel free to improve on the language here

Reply:
I updated the wording to be more clear, but I would like to keep the term `joining` since it's more faithful to whats happening. 'Correlating' is friendlier language but more vague